main.js

```diff
@@ -3,22 +3,41 @@ const storage = require("./storage/storageManager");
 const discord = require("discord.js");
 const client = new discord.Client();
 const config = require("./config");
-const cheerio = require("cheerio");
+const entreeBuilder = require("./entreeBuilder");
 
 client.on("ready", () => console.log("YES CHEF"));
 
+var onGoingEntrees = new Map();
+
 client.on("message", msg => {
   if (msg.author.bot) return;
 
-  if (msg.channel.id == "605862116808720416") {
-    let entry = parseEntry(msg);
-
-    if (entry != "bad") {
-      storage.write(entry);
+  if (onGoingEntrees.has(msg.author.id)) {
+    infoGather(onGoingEntrees.get(msg.author.id), msg.content);
+  }
 
-      msg.reply("your entry was added to the repldex");
-    }
+  if (msg.content === "?writeEntree") {
```

Comment:
What's going on if the user sends `?writeEntree` when he is in the info gathering phase ?

Reply:
I think there is not problem with the map, but is it clear for the user that the process restarted ?